celerybeatmongo/models.py

```diff
@@ -11,9 +11,14 @@
 
 
 def get_periodic_task_collection():
-    if hasattr(current_app.conf, "CELERY_MONGODB_SCHEDULER_COLLECTION") \
-            and current_app.conf.CELERY_MONGODB_SCHEDULER_COLLECTION:
-        return current_app.conf.CELERY_MONGODB_SCHEDULER_COLLECTION
+    if hasattr(current_app.conf, "mongodb_scheduler_collection") \
```

Comment:
I think this would be cleaner as 

```
if hasattr(current_app.conf, "mongodb_scheduler_collection"):
    return current_app.conf.get("mongodb_scheduler_collection")
elif hasattr(current_app.conf, "CELERY_MONGODB_SCHEDULER_COLLECTION"):
    return current_app.conf.get("mongodb_scheduler_collection")
```